api/src/main/java/com/thesurvey/api/util/UserUtil.java

```diff
@@ -41,6 +41,9 @@ public static String getUserNameFromAuthentication(Authentication authentication
     }
 
     public static void validateUserAuthentication(Authentication authentication) {
+        if (authentication == null) {
```

Comment:
Could you merge this in one `if statement` with the below one by like using `&&` operator?